fuseki/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM blankdots/jena-fuseki
+
+COPY config.ttl /data/fuseki/config/config.ttl
```

Comment:
Woo hoo. No more fuss with preparing fuseki: the datasets can be imperatively configured from a turtle file.

Reply:
I'd suggest mounting this as well

Reply:
Is it possible to mount individual files?

Reply:
Yes!